src/components/Toast/Toast.scss

```diff
@@ -1,59 +1,132 @@
 @import "../../styles/global-css-settings";
 @import "../../styles/typography";
+@import "../../styles/typography/typography";
 @import "../../styles/themes";
+@import "../../styles/theme-mixin";
 @import "../../styles/keyframes";
+@import "../../styles/typography/main-typography";
+@import "../../styles/typography/display-typography";
+
+$warning-background-color: linear-gradient(180deg, #fdfe53 0%, #eb9b3a 100%);
+$negative-background-color: linear-gradient(180deg, #e96878 0%, #d62e2e 100%);
+$toast-font-family: "Circular Std";
+$toast-border-radius: 12px;
+$toast-width: 343px;
+$toast-height: 72px;
 
 .l3-style-toast {
-  @include font-paragraph();
-  @include box-shadow-medium();
-  @include theme-prop(color, text-color-on-primary);
-  margin: $spacing-medium;
-  position: fixed;
-  right: 0;
-  left: 0;
-  top: 0;
-  margin-right: auto;
-  margin-left: auto;
-  padding: $spacing-small;
+  position: absolute;
+  width: $toast-width;
+  height: $toast-height;
   align-items: center;
   display: flex;
-  min-width: 200px;
-  border-radius: $border-radius-small;
+  flex-direction: row;
+  justify-content: center;
+  border-radius: $toast-border-radius;
 
-  &-icon {
-    display: flex;
-    padding-left: $spacing-small;
-    padding-right: $spacing-small;
+  &--position {
```

Comment:
this positioning will not work, the top left and the top right has the same styles try to fix it